src/Microsoft.AspNetCore.CookiePolicy/CookiePolicyMiddleware.cs

```diff
@@ -151,6 +151,22 @@ private void ApplyPolicy(CookieOptions options)
                     default:
                         throw new InvalidOperationException();
                 }
+                switch (Policy.SameSite)
+                {
+                    case SameSitePolicy.None:
+                        break;
+                    case SameSitePolicy.LaxOrStrict:
+                        if (options.SameSite == SameSiteEnforcementMode.None)
+                        {
+                            options.SameSite = SameSiteEnforcementMode.Lax;
+                        }
+                        break;
+                    case SameSitePolicy.AlwaysStrict:
+                        options.SameSite = SameSiteEnforcementMode.Strict;
+                        break;
+                    default:
+                        throw new InvalidOperationException();
```

Comment:
Poicy.SameSite.ToString(). This is more of an assert for if we add more enum values in the future.